### ISSUE TITLE
New Signature Request UI completely Freezes Metamask

### ISSUE THREAD

**Issue by BrendanChou** [MetaMask/metamask-extension#7395]
**Describe the bug**
The new Signature Request UI white-screens Metamask with no way to recover besides force-quitting the browser and starting over. I suspect the new UI is the issue as this works on v7.4.0.

**To Reproduce**
Send rpc request using the MetaMask-specific `eth_signTypedData_v4` rpcMethod.

**Expected behavior**
A signature request is shown.

**Screenshots**

**Browser details (please complete the following information):**
 - OSX
 - Chrome
 - MetaMask Version 7.5.0

**Comment by tothemoooooon** [comment]
Me too. This is blocking a launch of ours. Example repro (from the documentation) that works in the JS console:

```
// 1. enable

ethereum.enable();

// 2. sign

domain = [
    { name: "name", type: "string" },
    { name: "version", type: "string" },
    { name: "chainId", type: "uint256" },
    { name: "verifyingContract", type: "address" },
    { name: "salt", type: "bytes32" },
];
bid = [
    { name: "amount", type: "uint256" },
    { name: "bidder", type: "Identity" },
];
identity = [
    { name: "userId", type: "uint256" },
    { name: "wallet", type: "address" },
];
msg = {
  types: {
      EIP712Domain: domain,
      Bid: bid,
      Identity: identity,
  },
  domain: {
    name: "My amazing dApp",
    version: "2",
    chainId: parseInt(web3.version.network, 10),
    verifyingContract: "0x1C56346CD2A2Bf3202F771f50d3D14a367B48070",
    salt: "0xf2d857f4a3edcb9b78b4d503bfe733db1e3f6cdc2b7971ee739626c97e86a558"
  },
  primaryType: "Bid",
  message: {
    amount: 100,
    bidder: {
        userId: 323,
        wallet: "0x3333333333333333333333333333333333333333"
    }
  }
}
web3.currentProvider.sendAsync(
{
    method: "eth_signTypedData_v3",
    params: [ethereum.selectedAddress, JSON.stringify(msg)],
    from: ethereum.selectedAddress
},
function(err, result) {
  console.log('got ', err, result);
})
```

**Comment by danfinlay** [comment]
Can everyone experiencing this confirm they have the 7.5.0 update?

**Comment by tothemoooooon** [comment]
> Can everyone experiencing this confirm they have the 7.5.0 update?

Yes. 7.5.0. This was working yesterday and broke today, so I assume it was a recent rollout.

**Comment by tmashuang** [comment]
In @tothemoooooon's example we are expecting a `from` in the `message` so by replacing the message section to something like:
```
message: {
    from: {
      name: 'Account',
      wallet: '0x2222222222222222222222222222222222222222',
    },
    to: {
      amount: 100,
      bidder: {
          userId: 323,
          wallet: "0x3333333333333333333333333333333333333333"
      }
    }
  }
``` 
@BrendanChou Is your the same situation with the `message.from`, or was it a different section?

**Comment by danfinlay** [comment]
We have a rollback release blocked by Chrome review now, which we have to complete publishing before we can resume incrementally rolling out the branch with the hotfix.

**Comment by BrendanChou** [comment]
Looks fixed in 7.5.1, thanks all
